src/cli.rs

```diff
@@ -219,52 +216,42 @@ pub fn run() -> Result<()> {
     let conf = Config {
         agenix: agenix_conf,
         root: conf_path,
-        opts,
-        current_path,
     };
 
-    let opts = &conf.opts;
-
     trace!("rekey_all? {}", opts.rekey_all);
-    let paths = if opts.rekey_all {
-        let mut paths = HashSet::new();
-        for g in &conf.agenix.paths {
-            for p in glob::glob(&g.glob)
-                .wrap_err_with(|| format!("Failed to match glob pattern '{}'", &g.glob))?
+    if opts.rekey_all {
+        let mut paths = Vec::new();
```

Comment:
The main reason I used a `HashSet` instead of a `Vec` was to avoid rekeying a file multiple times in case it is matched by multiple patterns, but I just realized that this case isn't covered in the encryption/decrytion code either. How/if that should be handled is probably something for a later point in time. (Maybe a note about that could be added to the readme for the meantime.)

Reply:
Interesting. I opened a ticket at https://github.com/cole-h/agenix-cli/issues/8 -- it would be nice to have them all changed than one have different functionality than another.